tests/accuracy/test_crdt.py

```diff
@@ -45,53 +45,59 @@
 )
 
 LIN_DIR = join(dirname(__file__), os.pardir, "inputs", "crdt")
-RESULTS_PATH = join(dirname(__file__), os.pardir, "results", "crdt-test")
 
+ORDERS = ['coupling', 'sextupole', 'skewsextupole', 'octupole']
 
-def _create_input(order):
-    path_to_lin = join(LIN_DIR, order)
-    optics_opt = MEASURE_OPTICS_SETTINGS.copy()
-    optics_opt.update({
-        'files': [join(path_to_lin, f'{order}{idx}') for idx in range(1, 4)],
-        'outputdir': abspath(join(RESULTS_PATH, order)),
-    })
-    hole_in_one_entrypoint(**optics_opt)
-    return (optics_opt, path_to_lin)
 
+@pytest.fixture(scope='module')
+def _create_input():
```

Comment:
Could we not use `pytest`'s own [tmpdir or tmpdir_factory](https://docs.pytest.org/en/stable/tmpdir.html) fixture to make things cleaner?